apps/worker/package.json

```diff
@@ -89,7 +89,6 @@
     "typescript": "5.6.2"
   },
   "optionalDependencies": {
-    "@novu/ee-auth": "workspace:*",
```

Comment:
Worker doesn't need `@novu/ee-auth` as a dependency anymore (see changes in https://github.com/novuhq/packages-enterprise/pull/231)